staging_service/import_specifications/individual_parsers.py

```diff
@@ -316,3 +317,50 @@ def parse_excel(path: Path) -> ParseResults:
         return ParseResults(frozendict(results))
     else:
         return _error(Error(ErrorType.PARSE_FAIL, "No non-header data in file", spcsrc))
+
+
+def parse_dts_manifest(path: Path) -> ParseResults:
+    """
+    Parse the provided DTS manifest file. Expected to be JSON, and will fail otherwise.
+    The manifest should have roughly this format, with expected keys included:
+    {
+        "resources": [{
+            "id": str,
+            "name": str,
+            "path": str,
+            "format": str,
+            "instructions": {
+                "data_type": str,
+                "parameters": {
+                    "<param1>": value,
+                    "<param2>": value
+                }
+            }
+        }]
+    }
+    The parameters under the "instructions"."parameters" dictionary are arbitrary keys with
+    arbitrary values and will be returned as-is. They are expected to be PRIMITIVE_TYPEs and
+    will fail otherwise.
```

Comment:
This seems like it's expanding the import specification return format, which presumably the front end won't understand?

Reply:
Nope, it'll return the same format. I'll edit to clarify.

Reply:
:+1: